src/charm.py

```diff
@@ -317,6 +331,30 @@ def _build_pebble_layer(self) -> Layer:
 
         return pebble_layer
 
+    @property
+    def _node_exporter_pebble_layer(self) -> Layer:
+        """Construct the pebble layer.
+
+        Returns:
+            a Pebble layer specification for the Loki workload container.
+        """
+        pebble_layer = Layer(
+            {
+                "summary": "Node Exporter layer",
+                "description": "pebble config layer for Node Exporter",
+                "services": {
+                    "node-exporter": {
+                        "override": "replace",
+                        "summary": "node exporter",
+                        "command": "/bin/node_exporter",
```

Comment:
Should we only enable the disk collector for now?

Reply:
Do you think there would be a significant difference in performance?

Reply:
I was actually thinking about preventing alert rules to rely on the existence of metric names we may want to remove in the future.

Reply:
The collector we need is `filesystem`: _"Exposes filesystem statistics, such as disk space used."_

The include and exclude flags are mutually exclusive on collectors. The exclude flags are used to indicate "all except", while the include flags are used to say "none except".

The problem we have is that the filesystem collector does not have available `include` option: https://github.com/prometheus/node_exporter?tab=readme-ov-file#include--exclude-flags

Reply:
Well the documentation seems not to be as complete and clear as you may desire.... by reading the options provided in `node_exporter --help` I could figure out how to only enable the filesystem metrics:

`node_exporter --collector.disable-defaults --collector.filesystem`